runtime/manifest-parser.peg

```diff
@@ -86,16 +87,25 @@ Name
   }
 
 RecipeParticle
-  = ref:ParticleRef name:(whiteSpace Name)? eolWhiteSpace connections:(Indent (SameIndent RecipeParticleConnection)*)?
+  = ref:ParticleRef name:(whiteSpace Name)? eolWhiteSpace connections:(Indent (SameIndent (RecipeParticleConnection / RecipeParticleSlotConnection))*)?
```

Comment:
For consistency we should move RecipeParticleConnection / RecipeParticleSlotConnection into a new rule called RecipeParticleItem, distinguish them by 'kind' and disambiguate in Manifest.parse. But a TODO is OK for now.